gating/generate_release_notes/release_notes_dockerfile

```diff
@@ -9,9 +9,11 @@ RUN echo "jenkins ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 RUN apt-get install -y pandoc
 
 # use released versions of the differs when they are available.
-RUN pip install rpc_differ==0.3.0 reno==2.5.0
+RUN pip install rpc_differ==0.3.0 reno==2.5.1
 RUN pip install git+https://github.com/major/osa_differ@0.3.1
```

Comment:
Should we bump this to 0.3.2?

Reply:
That's a separate piece of work - https://rpc-openstack.atlassian.net/browse/RE-622